src/python/pants/engine/native.py

```diff
@@ -168,7 +168,7 @@
 Handle val_for(Key);
 
 Tasks* tasks_create(void);
-void tasks_task_begin(Tasks*, Function, TypeConstraint);
+void tasks_task_begin(Tasks*, Function, TypeConstraint, int);
```

Comment:
Rather than an `int`, this should be a `_Bool`: see the examples above.

I'll go ahead and make the edit directly here (you'll need to pull your branch).